tests/phpunit/data/wpml-config.xml

```diff
@@ -45,12 +45,23 @@
 				</key>
 				<key name="empty_option">
 					<key name="*">
-		            	<key name="label"/>
+						<key name="label"/>
 					</key>
 				</key>
 				<key name="simple_string_option" />
 				<key name="generic_option_*" />
 		</admin-texts>
+		<gutenberg-blocks>
+			<gutenberg-block type="core/image" translate="1">
+				<xpath>//figure/figcaption</xpath>
+				<xpath>//figure/img/@alt</xpath>
+			</gutenberg-block>
+		</gutenberg-blocks>
+		<gutenberg-blocks>
+			<gutenberg-block type="core/navigation-link" translate="1">
+				<key name="description" />
+			</gutenberg-block>
+		</gutenberg-blocks>
```

Comment:
Don't use core blocks.

Reply:
Done in 626270c464222444ed65a79ea783085253d34a17